kolibri/plugins/coach/assets/src/modules/examCreation/handlers.js

```diff
@@ -237,12 +236,12 @@ export function showPracticeQuizCreationPreviewPage(store, params) {
 function _prepPracticeQuizContentPreview(store, classId, contentId) {
   return ContentNodeResource.fetchModel({ id: contentId }).then(
     contentNode => {
-      const contentMetadata = assessmentMetaDataState(contentNode);
+      const contentMetadata = contentNode.assessmentmetadata || {};
```

Comment:
Just for clarity - could we update the variable name to `assessmentMetadata` rather than `contentMetadata`?